packages/builder/src/actions/rounds.ts

```diff
@@ -87,6 +87,11 @@ export const loadRound =
 
     const chainId = roundChainId || stateChainID;
 
+    // todo: remove or fix this
+    if (chainId === 5) {
```

Comment:
what does this do?

Reply:
This was for a weird bug on Goerli, which is depreciated now anyway. Removing for productions environment(s), we won't be using Goerli moving forward anyway.